mcs/class/Mono.Debugger.Soft/Test/dtest.cs

```diff
@@ -2384,73 +2384,52 @@ Event step_out_await (string method, Event e)
 	}
 
 	[Test]
-	// Broken by mcs+runtime changes (#5438)
-	[Category("NotWorking")]
 	public void LineNumbers () {
 		Event e = run_until ("line_numbers");
 
-		step_req = create_step (e);
-		step_req.Depth = StepDepth.Into;
-		step_req.Enable ();
+		create_step (e);
+		e = step_into ();
 
 		Location l;
-		
-		vm.Resume ();
 
-		e = GetNextEvent ();
-		Assert.IsTrue (e is StepEvent);
+		e = step_once ();
 
 		l = e.Thread.GetFrames ()[0].Location;
 
-		Assert.IsTrue (l.SourceFile.IndexOf ("dtest-app.cs") != -1);
+		Assert.AreEqual ("dtest-app.cs", Path.GetFileName (l.SourceFile));
 		Assert.AreEqual ("ln1", l.Method.Name);
-
-		// Check hash
-		using (FileStream fs = new FileStream (l.SourceFile, FileMode.Open, FileAccess.Read)) {
-			MD5 md5 = MD5.Create ();
```

Comment:
Why is this removed ?

Reply:
The test fails, and it doesn't have anything to do with line numbers. As the test has been disabled for 5+ years, and the MD5 is not tested anywhere else, I assume it is just broken. I will create another test for that and will leave it disabled.